src/example-types/results.js

```diff
@@ -106,7 +107,8 @@ module.exports = {
     return search(searchObj).then(results => ({
       scrollId: results._scroll_id,
       response: {
-        totalRecords: results.hits.total,
+        // ES 7+ is total.value, ES 6- is hits.total
+        totalRecords: F.getOrReturn('value', results.hits.total),
```

Comment:
This is the elasticsearch equivalent of duck typing 😅 

We may want to just drop ES 6 support since this is a breaking change, but this is one of only two things needed to maintain ES 6 support.